packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/method_channel/utils/source.dart

```diff
@@ -16,3 +17,13 @@ String getSourceString(Source source) {
       return 'default';
   }
 }
+
+
+/// Converts [AggregateSource] to [String]
+String getAggregateSourceString(AggregateSource source) {
+  switch (source) {
+    case AggregateSource.server:
+    default:
```

Comment:
I think we should remove the default here. It will help us catch missing case more easily with the linter. We have full control on this enum so no need to have fallback imo